### PR TITLE
[Requires Other PR] - Multiple Language Stutter

### PR DESCRIPTION
**Pull request by Kiyahitayika**
## Requires

PR: #302 

## What Does This PR Do
1.) Allows players to choose which language(s) they would like a character to stutter on in Character Setup. They may choose from:

- Galactic Common
- Species Language
- Secondary Language (if one has been selected)

2.) Recodes the Stutter Mechanic so that this new type of stutter does not interfere with the Damage Stutter type.
3.) Ensures that this Roleplaying Stutter Mechanic is not "curable". I didn't code a curing mechanism and it is *not* a part of Structural Enzymes, meaning it will not appear in the Admin Command Panel and it will not appear in the DNA Modifier in the Genetics Lab.

## Why It's Good For The Game
Resolving Issue: https://github.com/ScorpioStation/ScorpioStation/issues/215
Buffing Out Issue: https://github.com/ScorpioStation/ScorpioStation/issues/274

## Images of changes
Bingo. Bango.
https://cdn.discordapp.com/attachments/709188230817513633/792532654649114634/unknown.png
Blingo. Blango.
https://user-images.githubusercontent.com/69871346/102839786-4df31680-43cf-11eb-93ad-ee505510919f.mp4
Blinko. Blanko.
https://cdn.discordapp.com/attachments/709188230817513633/792839249857609748/unknown.png
Blonko. Blongo.
https://imgur.com/a/HPRi8cs

## Changelog
:cl:
add: Multiple Language Stutter
tweak: Stutter Mechanic
/:cl:

### CODE REVIEW

**Comment by Kiyahitayika** [comment]
I changed three less files in this PR versus #290 . What did I miss?

File: code/modules/mob/living/carbon/human/update_icons.dm [hunk 1]
```diff
@@ -450,7 +450,7 @@ GLOBAL_LIST_EMPTY(damage_icon_parts)
 	if(gender == FEMALE)
 		g = "f"
 	// DNA2 - Drawing underlays.
-	for(var/datum/dna/gene/gene in GLOB.dna_genes)
+	for(var/datum/dna/gene/gene in GLOB.all_dna_genes)
```
Comment (Kiyahitayika) [inline code/modules/mob/living/carbon/human/update_icons.dm:453]: This is wrong.

**Comment by Kiyahitayika** [comment]
@blinkdog Added the Code Needed flag as I am not sure how to make this interface with the SQL database. You and @ZomgPonies seem to have a very good handle on SQL - any advice/help is appreciated

**Comment by blinkdog** [comment]
> @blinkdog Added the Code Needed flag as I am not sure how to make this interface with the SQL database. You seem to have a very good handle on SQL - any advice/help is appreciated

Interfacing with SQL on this one is roughly the same as the other one.
Let me know if you have any questions specific to this PR?

(And don't forget the SQL tag! :smile:)

File: code/modules/mob/living/carbon/human/update_icons.dm (hunk 1 above)
Comment (Kiyahitayika) [reply code/modules/mob/living/carbon/human/update_icons.dm]: Not, it was right. You skrek!!!!!

File: code/__DEFINES/genetics.dm [hunk 2]
```diff
@@ -14,12 +14,16 @@
...
-#define DISABILITY_FLAG_LISP        256
-#define DISABILITY_FLAG_DIZZY       512
-#define DISABILITY_FLAG_CHAV        1024
-#define DISABILITY_FLAG_DEAF        2048
+#define DISABILITY_FLAG_SWEDISH     64
+#define DISABILITY_FLAG_LISP        128
+#define DISABILITY_FLAG_DIZZY       256
+#define DISABILITY_FLAG_CHAV        512
+#define DISABILITY_FLAG_DEAF        1024
+
```
Comment (Kiyahitayika) [inline code/__DEFINES/genetics.dm:22]: Missing 2048

**Comment by Kiyahitayika** [comment]
> The database schema needs a new column to contain the value. You can find the column for disabilities here:
> https://github.com/ScorpioStation/ScorpioStation/blob/master/SQL/scorpio/scorpio_schema.sql#L68
> A clone of that field (copy the line, paste it right below) with a new name (rename it on the next line) should work fine.

@blinkdog 

If I am understanding this correctly, because the Multiple Language Stutter bitflags are being set onto the Disabilities variable *and* the Disabilities variable is already being saved to the SQL database, the only thing I need to change is the size of the `mediumint`, right?

**Comment by Kiyahitayika** [comment]
Closing in preparation for upstream merge and merge of #302

**Pull request closed without merging**